int16 signal

float64 next_ref_x
float64 next_ref_y
float64 next_ref_z
